packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -158,7 +158,7 @@ class GuildBanManager extends CachedManager {
       reason: options.reason,
     });
     if (user instanceof GuildMember) return user;
-    const _user = this.client.users.resolve(id);
+    const _user = this.client.users.cache.get(id) ?? null;
```

Comment:
This is one of the cases, like in `packages/discord.js/src/client/actions/ThreadListSync.js`, where you don't need `?? null`